package-lock.json

```diff
@@ -1,3 +1,48 @@
 {
-  "lockfileVersion": 1
+  "requires": true,
```

Comment:
Were there updates to a package.json?

Reply:
I dont think so, i just ran an npm install after pulling from dev